tests/test_playutils_settings.py

```diff
@@ -0,0 +1,393 @@
+from unittest.mock import patch
+
+import pytest
+
+
+@pytest.fixture
+def mock_play_utils():
+    with patch("jellyfin_kodi.helper.playutils.PlayUtils") as MockPlayUtils:
+        mock_instance = MockPlayUtils.return_value
+        mock_instance.get_transcoding_video_codec.return_value = ""
+        yield mock_instance
+
+
+def test_hevc_transcoding_scenario(mock_play_utils):
```

Comment:
> ## [Codecov](https://app.codecov.io/gh/jellyfin/jellyfin-kodi/pull/895?dropdown=coverage&src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=jellyfin) Report
> Attention: Patch coverage is `0%` with `7 lines` in your changes missing coverage. Please review.
> 
> > Project coverage is 21.49%. Comparing base [(`62a1de7`)](https://app.codecov.io/gh/jellyfin/jellyfin-kodi/commit/62a1de7879c37accca1724979095d569eb225b51?dropdown=coverage&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=jellyfin) to head [(`ca49333`)](https://app.codecov.io/gh/jellyfin/jellyfin-kodi/commit/ca493338877092c4a46fbf40fc75b2fc636a7a9b?dropdown=coverage&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=jellyfin).
> > Report is 20 commits behind head on master.
> 
> [Files](https://app.codecov.io/gh/jellyfin/jellyfin-kodi/pull/895?dropdown=coverage&src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=jellyfin)	Patch %	Lines
> [jellyfin_kodi/helper/playutils.py](https://app.codecov.io/gh/jellyfin/jellyfin-kodi/pull/895?src=pr&el=tree&filepath=jellyfin_kodi%2Fhelper%2Fplayutils.py&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=jellyfin#diff-amVsbHlmaW5fa29kaS9oZWxwZXIvcGxheXV0aWxzLnB5)	0.00%	[7 Missing ⚠️ ](https://app.codecov.io/gh/jellyfin/jellyfin-kodi/pull/895?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=jellyfin)
> Additional details and impacted files
> [☔ View full report in Codecov by Sentry](https://app.codecov.io/gh/jellyfin/jellyfin-kodi/pull/895?dropdown=coverage&src=pr&el=continue&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=jellyfin). 📢 Have feedback on the report? [Share it here](https://about.codecov.io/codecov-pr-comment-feedback/?utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=jellyfin).

I'm not sure the coverage is calculated correctly. Or maybe it's based on filenames?
Added a lot of tests for these settings based methods and the issue itself.

Reply:
I am not familiar with mock yet, but it looks to me like the real function isn't actually getting exercised at all.

Is perhaps the whole function getting mocked here?
I think it is the settings function that should be mocked – `jellyfin_kodi.helper.utils.settings`, allowing the actual function to run, so that it can be tested.

That being said, I don't expect any sort of testing for modules interacting with the settings or other Kodi functionality *yet*.

Those particular functions can however easily be made into staticmethods (or classmethod), allowing them to be called without instantiating the class, making them easier to test (not referencing self nor the class itself).


There are two acceptable ways of fixing this for the PR:
- Removing the tests
- Fixing the tests to run against the actual code

If you wish to try your hand at fixing the tests, you should install the dev dependencies, and utilize the following commands to check the coverage yourself before committing the changes:
```bash
coverage run
coverage html
```
This will create a directory named `htmlcov/`, which contains a full test coverage report locally.